beets/util/pipeline.py

```diff
@@ -225,6 +225,18 @@ def abort_all(self, exc_info):
         for thread in self.all_threads:
             thread.abort()
 
+    def run(self):
```

Comment:
This (and `_run` in the subclasses) could use some docstrings to explain their new relationship.